doc/Quick start.md

```diff
@@ -115,7 +115,14 @@ export class Login {
     }
 
     login() {
-        return this.authService.login(this.email, this.password)
+        return this.authService.login({email: this.email, password: this.password})
+        // // or for eg OpenIddict
```

Comment:
Maybe put this elsewhere. A quick-start is an opinionated guide to get someone into the plugin quickly. Anything specific belongs on a different page.